src/strategies/poap-with-weight/examples.json

```
@@ -0,0 +1,22 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "poap-with-weight",
+      "params": {
+        "address": "0x22C1f6050E56d2876009903609a2cC3fEf83B415",
+        "symbol": "POAP",
+        "ids": ["613607", "613237"]
```

you are using `tokenIds` in strategy code, which is throwing `undefined` error
```suggestion
        "tokenIds": ["613607", "613237"]
```